includes/block.php

```diff
@@ -474,25 +475,21 @@ function register_meta_endpoint() {
 		'content-aggregator-block/v1',
 		'meta',
 		array(
-			'methods'  => 'GET',
-			'callback' => __NAMESPACE__ . '\meta_rest_response',
+			'methods'             => 'GET',
+			'callback'            => __NAMESPACE__ . '\meta_rest_response',
+			'permission_callback' => '__return_true',
 		)
 	);
 }
 
 /**
- * Return the meta keys registered for the provided post type.
- *
- * @param \WP_Request $request The incoming REST API request object.
- *
- * @return array Posts found using the provided parameters.
+ * Make sure only users who can edit posts can access this REST API route.
  */
-function meta_rest_response( $request ) {
```

Comment:
It looks like we should keep this function and remove `meta_rest_permission()` instead.

Reply:
Whoops! For sure.